OasisDEX/oasis-borrow
# Ajna borrow and earn history

# [UI components](https://app.shortcut.com/oazo-apps/story/9553/ajna-history-ui-components)
# [Datasorce](https://app.shortcut.com/oazo-apps/story/9552/ajna-history-map-front-end-to-subgraph-events)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added set of components to handle history
- addded mappers to handle new history event interface
- handled borrow and earn ajna history
  
## How to test 🧪
  <Please explain how to test your changes>

- on relatively new Ajna position go to history tab and verify whether everything looks ok (except for LTV and fees due to lack of tokens prices on goerli)
